### PR TITLE
[cli-sessions]

### PR DESCRIPTION
**Pull request by i-davydenko**
Add support for cli-sessions feature

What I did:
Added handlers for new SSH_SERVER and SERIAL_CONSOLE attributes.

Why I did it:

Give ability to:
configure limit for active login sessions.
configure ssh-server / serial console autologout timeout
configure sysrq-capabilities (enable / disable)

How I did it

Add new service that responsible for serial configuration;
Update existing flows for extended ssh-server configurations in hostcfgd;
Add YANG model to support new configuration.

How I verified it:

Configure, reconfigure, and reset all new parameters, check if applicapple parameters was updated in the system.
Added unittests.

Signed-off-by: Ivan Davydenko [ivanda@nvidia.com](mailto:ivanda@nvidia.com)

### CODE REVIEW

**Comment by qiluo-msft** [comment]
Could you refine a meaningful PR title?

File: scripts/hostcfgd [hunk 1]
```diff
@@ -1128,16 +1139,31 @@ class PamLimitsCfg(object):
...
+        if "localhost" not in device_metadata and "POLICIES" not in ssh_server_policies:
             return
 
         self.read_localhost_config(device_metadata["localhost"])
+        self.read_max_sessions_config(ssh_server_policies.get("POLICIES", None))
         self.render_conf_file()
 
+    # Read max_sessions config
+    def read_max_sessions_config(self, ssh_server_policies):
+        if ssh_server_policies is not None:
```
Comment (liuh-80) [inline scripts/hostcfgd:1163]: if not ssh_server_policies:

Comment (i-davydenko) [reply scripts/hostcfgd:1357]: this will break logic :(

**Comment by liat-grozovik** [comment]
@i-davydenko please handle missing coverage

**Comment by i-davydenko** [comment]
> @i-davydenko please handle missing coverage

Done

**Comment by qiluo-msft** [comment]
@liuh-80 do you have more concern?

**Review by liuh-80 — APPROVED** [review]

**Comment by liuh-80** [comment]
> @liuh-80 do you have more concern?

No, Approved

**Review by qiluo-msft — APPROVED** [review]

**Pull request merged**